src/shared_modules/utils/timeHelper.h

```diff
@@ -22,21 +22,24 @@ namespace Utils
 {
     static std::string getTimestamp(std::time_t time)
     {
-        std::string timestamp;
-        tm* localTime { localtime(&time) };
-        // Final timestamp: "YYYY/MM/DD h:m:s"
-        timestamp  = std::to_string(localTime->tm_year + 1900);
-        timestamp += "/";
-        timestamp += std::to_string(localTime->tm_mon + 1);
-        timestamp += "/";
-        timestamp += std::to_string(localTime->tm_mday);
-        timestamp += " ";
-        timestamp += std::to_string(localTime->tm_hour);
-        timestamp += ":";
-        timestamp += std::to_string(localTime->tm_min);
-        timestamp += ":";
-        timestamp += std::to_string(localTime->tm_sec);
-        return timestamp;
+        std::stringstream ss;
+        // gmtime: result expressed as a UTC time
+        tm* localTime { gmtime(&time) };
+        // Final timestamp: "YYYY/MM/DD hh:mm:ss"
+        // Date
+        ss << std::hex << std::setfill('0') << std::setw(4) << std::to_string(localTime->tm_year + 1900);
```

Comment:
is necessary the std::hex here ?

Reply:
It is not. Removed in latest commit. Good catch